### PR TITLE
feat: Add web dockerfile.

### LINKED ISSUE #79: Feature: Add Dockerfiles for web.
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've discussed this feature request in the [OpenIMSDK Slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) and got positive feedback

### Is this feature request related to a problem?

❎ No

### Problem Description

Support `OpenKF` running in docker clusters in first stage.

### Solution Description

1. Add web docker image in `/web`.
2. Preparing for static deployment.
3. Push image in ghcr.

### Benefits

Easy to deploy system.

### Potential Drawbacks

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by IRONICBo**
#### 🔍 What type of PR is this?

/kind feature

#### 👀 What this PR does / why we need it:

Add docker image for web.

#### 🅰 Which issue(s) this PR fixes:

Fixes #79

### CODE REVIEW

**Comment by IRONICBo** [comment]
The signature to be committed in order to sign openim-sigs the CLA

**Comment by IRONICBo** [comment]
recheck

File: deploy/docker/Dockerfile.web [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# Build stage
+FROM node:18 AS builder 
+
+WORKDIR /openkf-web
+
+COPY /web .
+RUN npm install
+RUN chmod +x scripts/setup.sh && ./scripts/setup.sh
+RUN npm run build
+
```
Comment (charles-chenzz) [inline deploy/docker/Dockerfile.web:24]: RUN npm install \
&& RUN chmod +x scripts/setup.sh && ./scripts/setup.sh \
&& RUN npm run build

no need to separate it to three layers

File: server/config.example.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  port: 10010
+
+jwt:
+  secret: openkf
+  issuer: openkf
+  expire_days: 1
+
+mysql:
+  ip: 127.0.0.1
+  port: 3307
```
Comment (charles-chenzz) [inline server/config.example.yaml:31]: we use 3307 here? I remember mysql default port is 3306

Comment (cubxxw) [reply server/config.example.yaml:31]: The agreed 3307

**Review by cubxxw — CHANGES_REQUESTED** [review]
lgtm

File: deploy/docker/Dockerfile.web (hunk 1 above)
Comment (IRONICBo) [reply deploy/docker/Dockerfile.web:24]: Thanks!

File: server/config.example.yaml (hunk 2 above)
Comment (IRONICBo) [reply server/config.example.yaml:31]: We're forwarding ports here.

**Review by cubxxw — APPROVED** [review]

**Comment by cubxxw** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by IRONICBo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by IRONICBo** [comment]
recheck

**Pull request merged**